Cargo.toml

```diff
@@ -178,3 +178,4 @@ sha2 = { git = "https://github.com/risc0/RustCrypto-hashes", tag = "sha2-v0.10.8
 # crypto-bigint = { git = "https://github.com/risc0/RustCrypto-crypto-bigint", tag = "v0.5.5-risczero.0" }
 secp256k1_v028 = { package = "secp256k1", version = "0.28", git = "https://github.com/Sovereign-Labs/rust-secp256k1.git", branch = "risc0-compatible-0-28-2" }
 k256 = { package = "k256", version = "0.13.3", git = "https://github.com/risc0/RustCrypto-elliptic-curves", tag = "k256/v0.13.3-risczero.0" }
+bitcoincore-rpc = { version = "0.18.0", git = "https://github.com/jfldde/rust-bitcoincore-rpc.git" }
```

Comment:
my man... don't tell me you are trying to take over the world by installing malware from your local repository.